### PR TITLE
Fix examples showing the weak password banners

### PR DESCRIPTION
**Pull request by kwart**
We don't have the Group password anymore and the symmetric encryption feature is deprecated. Let's show up-to-date examples.

### CODE REVIEW

File: docs/modules/security/pages/validating-secrets.adoc [hunk 1]
```diff
@@ -87,29 +91,33 @@ above system property is set to “true”:
 
 [source,shell,options="nowrap"]
```
Comment (JakeSCahill) [inline docs/modules/security/pages/validating-secrets.adoc:92]:
```suggestion
[source,shell,subs="attributes+",options="nowrap"]
```

File: docs/modules/security/pages/validating-secrets.adoc [hunk 2]
```diff
@@ -87,29 +91,33 @@ above system property is set to “true”:
...
 ----
-WARNING: [192.168.2.112]:5701 [dev] [4.0-SNAPSHOT]
+WARNING: [10.8.0.10]:5701 [dev] [5.2.0]
```
Comment (JakeSCahill) [inline docs/modules/security/pages/validating-secrets.adoc:94]:
```suggestion
WARNING: [10.8.0.10]:5701 [dev] [{full-version}] 
```

File: docs/modules/security/pages/validating-secrets.adoc [hunk 3]
```diff
@@ -87,29 +91,33 @@ above system property is set to “true”:
...
-at com.hazelcast.instance.HazelcastInstanceImpl.createNode(HazelcastInstanceImpl.java:163)
-at com.hazelcast.instance.HazelcastInstanceImpl.<init>(HazelcastInstanceImpl.java:130)
-at com.hazelcast.instance.HazelcastInstanceFactory.constructHazelcastInstance(HazelcastInstanceFactory.java:195)
-at com.hazelcast.instance.HazelcastInstanceFactory.newHazelcastInstance(HazelcastInstanceFactory.java:174)
-at com.hazelcast.instance.HazelcastInstanceFactory.newHazelcastInstance(HazelcastInstanceFactory.java:124)
-at com.hazelcast.core.Hazelcast.newHazelcastInstance(Hazelcast.java:58)
-----
\ No newline at end of file
+Oct 10, 2022 11:00:52 AM com.hazelcast.instance.impl.Node
+SEVERE: [10.8.0.10]:5701 [dev] [5.2.0] Node creation failed
```
Comment (JakeSCahill) [inline docs/modules/security/pages/validating-secrets.adoc:110]:
```suggestion
SEVERE: [10.8.0.10]:5701 [dev] [{full-version}] Node creation failed
```

File: docs/modules/security/pages/validating-secrets.adoc (hunk 2 above)
Comment (JakeSCahill) [reply docs/modules/security/pages/validating-secrets.adoc:94]: We can make sure this displays the same version as the docs.

**Comment by kwart** [comment]
Thanks, Jake, for the review and the fixes.

**Review by JakeSCahill — APPROVED** [review]

**Comment by JakeSCahill** [comment]
@kwart does this fix apply back to 5.0?

**Comment by kwart** [comment]
> @kwart does this fix apply back to 5.0?

even back to `4.0` :-)

**Pull request merged**